nics-tw/PETsARD
# bug: fix custom_data parameter assignment

- bug
    - PETsARD/loader/splitter.py
        - assign parameter for Loader in Splitter custom_data - 8264924417a6efddb434bacb7d9c3370c1c5b2e0 
    - PETsARD/synthesizer/synthesizer.py
        - assign parameter for Loader in Synthesizer custom_data - 578c891e62a9d8b475abcf0fa88ba70e7b290c88 
    - PETsARD/util/optimize_dtypes.py
        - fix optimize_dtype() under all data in column are nan - 1f4ba450d073f4c5c36a8365a7c408613a037ec2 

- test
    - demo/dev/Issue554.ipynb
        - create Issue554.ipynb - 0592ccda41964a6645b25886f9800cd5baf6c490 
        - update test case - 1f530fdde6bddf3db331aef09001bfae1d5f11e8 
        - remove Issue554.ipynb - 3c6cdba1e1f3d72f47a5e8318505a0603d4f558a 

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1207354312935984
